lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -0,0 +1,15 @@
+(function() {
+  <% appsjs.each do |appjs| %>
+    <%= appjs %>
+  <% end %>
+
+  <% installations.each do |installation| %>
+    ZendeskApps[<%= installation.app.name.to_json %>].install(<%= installation.to_json %>);
+  <% end %>
+
+  <% installation_orders.each do |loc_name, installation_list| %>
+    ZendeskApps.sortAppsForSite(<%= loc_name.to_json %>, <%= installation_list.to_json %>);
+  <% end %>
+}());
+
+ZendeskApps.trigger && ZendeskApps.trigger('ready');
```

Comment:
Not sure we need the guard here anymore, `ZendeskApps` will always be an event emitter now.

`ZendeskApps.trigger('ready')` should suffice.